uui/components/tables/columnsConfigurationModal/ColumnsConfigurationModal.tsx

```diff
@@ -73,7 +73,21 @@ export function ColumnsConfigurationModal<TItem, TId, TFilter>(props: ColumnsCon
                 {!!amountPinned && (
                     <FlexRow cx={ styles.groupItems } size="30">
                         {groupedColumns.displayedPinned.map((c) => (
-                            <ColumnRow column={ c } key={ c.key } />
+                            <ColumnRow
+                                renderItem={ () => (
+                                    <FlexRow>
```

Comment:
Why we need it here?  Is it temporary code for debug?

Reply:
Yes, it was for debugging.